### PR TITLE
add: Avatar Component

### PR DESCRIPTION
**Pull request by Varu98**
Screenshot (8)
Screenshot (7)

### CODE REVIEW

**Review by vijaytembugade — APPROVED** [review]
Looks good Vardaan. Nice variants.

**Comment by Varu98** [comment]
> Looks good Vardaan. Nice variants.

Thanks Vijay! 😄

File: styles/avatar.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+.online-div {
+  position: absolute;
+  bottom: -10%;
+  right: -10%;
+  border-radius: 50%;
+  border: 1px solid #fff;
+  background-color: greenyellow;
+  width: 1.5rem;
+  height: 1.5rem;
+  /* box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.3); */
```
Comment (nshanbhag20001412) [inline styles/avatar.css:52]: we can remove the commented codes if not required

**Review by nshanbhag20001412 — APPROVED** [review]

File: styles/avatar.css (hunk 1 above)
Comment (nshanbhag20001412) [reply styles/avatar.css:52]: rest looks great

**Pull request merged**